.github/workflows/build-documentation.yml

```diff
@@ -49,6 +58,10 @@ jobs:
           root_file: hydras.tex
           working_directory: doc
           latexmk_use_lualatex: true
+          #pre_compile: |
+          #  apk update
+          #  yes | apk add py3-pip
+          #  yes | pip3 install alectryon
 
       - name: Extract PDF and Coq-Doc
         if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
We build the PDF and coqdoc in all cases and we should extract them in all cases, so that they are made available as an artifact in pull requests:
```suggestion
```